src/node.cc

```diff
@@ -3711,6 +3716,7 @@ static void PrintHelp() {
          "                             note: linked-in ICU data is present\n"
 #endif
          "  --preserve-symlinks        preserve symbolic links when resolving\n"
+         "  --experimental-modules     experimental ES Module support\n"
```

Comment:
perhaps `--expose-esm` to keep it consistent with the `--expose-http2`, `--expose-internals`, etc? Not critical tho